extensions/amp-install-serviceworker/amp-install-serviceworker.md

```diff
@@ -73,6 +73,65 @@ URL of a HTML document that install a ServiceWorker.
 
 Must have the value `nodisplay`.
 
+### `data-no-service-worker-fallback-url-match`
+
+A regular expression that matches URLs to be rewritten to navigate via shell for no-service-worker fallback.
+See [Shell URL rewrite](#shell-url-rewrite) section for more details.
+
+The value must be a valid JavaScript RegExp string.
```

Comment:
Done.

I do currently test against the complete URL, including origin and query string. Most of masks are so simple that this is not an issue. Do you think I should only test against path?